### PR TITLE
[SYCL] Disable Signbit issue for NAN

### PR DESCRIPTION
**Pull request by jinge90**
Disabled test cases impacted by the fact that Clang and MSVC compilers have different behavior for sign bit of NAN/-NAN.

Signed-off-by: jinge90 <ge.jin@intel.com>

### CODE REVIEW

**Comment by jinge90** [comment]
Hi, @vladimirlaz 
It seems that clang-format tool crashed when running format check for this patch, is it a know issue?

Thanks very much.

File: SYCL/DeviceLib/cmath_fp64_test.cpp [hunk 1]
```diff
@@ -97,8 +95,8 @@ template <class T> void device_cmath_test(s::queue &deviceQueue) {
 
         res_access[i++] = !(std::signbit(infinity) == 0);
         res_access[i++] = !(std::signbit(minus_infinity) != 0);
-        res_access[i++] = !(std::signbit(nan) == 0);
-        res_access[i++] = !(std::signbit(minus_nan) != 0);
+        // res_access[i++] = !(std::signbit(nan) == 0);
+        // res_access[i++] = !(std::signbit(minus_nan) != 0);
```
Comment (vladimirlaz) [inline SYCL/DeviceLib/cmath_fp64_test.cpp]: If the code is not needed it is better to delete it

**Review by vladimirlaz — CHANGES_REQUESTED** [review]
Could you apply clang-format locally?

File: SYCL/DeviceLib/cmath_fp64_test.cpp (hunk 1 above)
Comment (jinge90) [reply SYCL/DeviceLib/cmath_fp64_test.cpp]: Done.

**Comment by jinge90** [comment]
> Could you apply clang-format locally?

Hi, @vladimirlaz 
I have applied clang-format locally but the patch still lead to clang-format crash in pre-ci environment.

I just used "clang-format-9 cmath_fp64_test.cpp" to get the clang formatted patch.

Thanks very much.

**Comment by jinge90** [comment]
Hi, @intel/llvm-reviewers-runtime @vladimirlaz @vzakhari 
Could you help review this patch? We need it to be merged as soon as possible.
Thanks very much.

**Comment by vladimirlaz** [comment]
@jinge90 it looks like test coverage has been reduced. Could you please provide motivation in the PR description?

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**